### PR TITLE
Edit and Save speaker bio

### PR DESCRIPTION
**Pull request by jebbinSCL**
Note: Obviously there is repetition here, but I'm waiting until I've added a few more editing controls before I refactor.

### CODE REVIEW

File: app/actions/index.js [hunk 1]
```diff
@@ -63,3 +69,12 @@ export const updateSpeakerRating = (speakerId, newRating) =>
...
     };
+
+export const updateSpeakerBio = (speakerId, newBio) =>
+    (dispatch) => {
+        dispatch(createAction(actionTypes.UPDATE_SPEAKER_BIO_START)());
+        return SpeakersService.updateBio(speakerId, newBio)
+            .then(() => dispatch(createAction(actionTypes.UPDATE_SPEAKER_BIO_COMPLETE)(newBio)))
+            .then(() => dispatch(changeSpeakerBioEditMode(false)))
+            .catch(error => dispatch(createAction(actionTypes.UPDATE_SPEAKER_BIO_ERROR)(error)));
+    };
```
Comment (pallenSL) [inline app/actions/index.js]: Is there a reason we are sometimes calling createAction() directly and sometimes defining a method to do it and calling that?

File: app/components/Session/EditSaveControl.jsx [hunk 2]
```diff
@@ -51,10 +51,10 @@ class EditSaveControl extends Component {
 }
 
 EditSaveControl.propTypes = {
-    children: PropTypes.element.isRequired,
-    changeEditMode: PropTypes.func,
-    onSaveClick: PropTypes.func,
-    inEditMode: PropTypes.bool,
+    children: React.PropTypes.oneOfType([PropTypes.element, PropTypes.string]).isRequired,
```
Comment (pallenSL) [inline app/components/Session/EditSaveControl.jsx]: Why do we need "React." on this one?

File: app/actions/index.js (hunk 1 above)
Comment (jebbinSCL) [inline app/actions/index.js]: All of the uses of createAction directly are simply propagating data in memory to the store, while the larger functions call createAction and do other work as well. They are also asynchronous. This is where the call to the services starts.

File: app/components/Session/EditSaveControl.jsx (hunk 2 above)
Comment (jebbinSCL) [inline app/components/Session/EditSaveControl.jsx]: That was a mistake, thanks.

Comment (jebbinSCL) [inline app/components/Session/EditSaveControl.jsx]: Adjusted

**Comment by pallenSL** [comment]
Shippety ship it!
:shipit:

**Pull request merged**